doc/gettingstarted.md

```diff
@@ -268,6 +275,24 @@ Once the downlink has been acknowledged, a message is published to the topic `v3
 }
 ```
 
+Multicast class C downlinks can be scheduled by adding the `class_bc` flag to the downlink, which specifies on which gateways the downlink should be scheduled.
```

Comment:
Let's put this in a separate section and refer explicitly to the `--supports-class-c` flag